src/components/sbb-header/readme.md

```diff
@@ -0,0 +1,110 @@
+# sbb-header
+
+The `sbb-header` component is a container for actions and logo, and it is displayed sticky at page's top.
+
+It has two slots: 
+the first one can contain one or more [sbb-header-action](../sbb-header-action/readme.md) or other action items
+like `sbb-button` or `sbb-link`, and it is displayed at the left end of the component; the second slot is displayed
+at the right end, and it can contain a logo, which by default is the [sbb-logo](../sbb-logo/readme.md).
+
+A box-shadow appears under the component if the `shadow` variable is set to `true`. 
```

Comment:
Could you add documentation that this should be done when scrolled and removed when the scroll position is at the top?

Reply:
Fixed with commit [ebe6bba](https://github.com/lyne-design-system/lyne-components/pull/1357/commits/ebe6bba4e15332000e76b8d5a616faf250870923).